### ISSUE TITLE
Problem with create vm from save_as img

### ISSUE THREAD

**Issue by leagueofsoups** [OpenNebula/one#5835]
**Description**
I try prepare base image for my test environment and use save_as button on web interface.
![image](https://user-images.githubusercontent.com/41836635/169710920-61b65827-6e9a-4bda-8110-38a12d35d6f8.png)
When I try create new VM and use saved image, I have error
![image](https://user-images.githubusercontent.com/41836635/169710985-8f65873d-197d-4f6e-a90c-53255ce84752.png)

**To Reproduce**
Try to use saved image.

**Details**
 - Hypervisor: KVM
 - Version: 6.2.0

**Additional context**
oneimage show <save_as img id> --xml | grep -i FORMAT
```
  <FORMAT><![CDATA[save_as]]></FORMAT>
```

Problem inside domain xml:
cat /var/lib/one//datastores/0/378/deployment.0 | grep "save_as"
```
<driver name='qemu' type='save_as' cache='none' discard='unmap'/>
```

This is look like proble [hear](https://github.com/OpenNebula/one/blob/63eb2447db602a964029441a6b77ad7735a953d2/src/image/Image.cc#L179)

**Comment by leagueofsoups** [comment]
![image](https://user-images.githubusercontent.com/41836635/170554501-caf71f5e-33cc-477a-8af7-7f0549aa5e98.png)
Resolved after remove this attributes from base image

**Comment by al3xhh** [comment]
This issue is already solved on 6.4 version, check this commit 8a875e88880e333e1802f0237ba69bbaefa2247c. Closing it.
